src/system-health/health_checker/sysmonitor.py

```diff
@@ -466,5 +466,22 @@ def task_worker(self):
             return
         self.system_service()
 
+    def task_stop(self):
+        # Signal the process to stop
+        self.task_stopping_event.set()
+        #Clear the resources of mpmgr- Queue
+        self.mpmgr.shutdown()
+
+        # Wait for the process to exit
+        self._task_process.join(self._stop_timeout_secs)
```

Comment:
should monitor_system_bus task and monitor_statedb_table task be join here?

Reply:
At the end of the system_service() function, there exists a call to monitor_system_bus.task_stop() and monitor_statedb_table.task_stop(), which are responsible for shutting down their respective tasks.